### PR TITLE
testeth: add stTimeConsuming

### PR DESCRIPTION
**Pull request by winsvega**
this tests take more then 3 minutes of test execution time. 
also PR updates the test submodule head

### CODE REVIEW

File: test/tools/jsontests/StateTests.cpp [hunk 1]
```diff
@@ -147,7 +147,8 @@ class GeneralTestFixture
         boost::filesystem::path suiteFillerPath = suite.getFullPathFiller(casename).parent_path();
 
         // Check specific test cases
-        static vector<string> timeConsumingTestSuites{string{"stQuadraticComplexityTest"}};
+        static vector<string> timeConsumingTestSuites{
```
Comment (gumb0) [inline test/tools/jsontests/StateTests.cpp]: can be const

**Comment by gumb0** [comment]
cc @chfast

**Comment by chfast** [comment]
I don't understand the direction this is going. Is it safe to skip these tests on regular basis? I'd rather check why they take so much time, maybe we can reduce the number of them, etc. If you disable them, 99% of users (including us) will not run them. 

If you moved e.g. stSstore tests, does it mean we don't test (unless explicitly enabled) one of the features at all?

**Comment by winsvega** [comment]
Only those tests I moved which take a lot of time to execute.

**Comment by chfast** [comment]
> Only those tests I moved which take a lot of time to execute.

You mean test suites or test cases?

**Comment by winsvega** [comment]
Test cases. One specific test is 50000 sha256 execution. 
And 5000 of sstore combinations tests

**Comment by chfast** [comment]
Ok, that's fine then.

> One specific test is 50000 sha256 execution.

Are you sure such tests are useful? I believe you get the same effect when running sha256 10x instead of 50000.

**Comment by winsvega** [comment]
> Are you sure such tests are useful? I believe you get the same effect when running sha256 10x instead of 50000.

don't remember why exactly Christoph added this test case. but I see its potential to crush some clients or make it timeout due to not optimized coding.

**Comment by winsvega** [comment]
net tests failing randomly

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,9 @@
 - Added: [#5557](https://github.com/ethereum/aleth/pull/5557) Improved debug logging of full sync.
 - Added: [#5564](https://github.com/ethereum/aleth/pull/5564) Improved help output of Aleth by adding list of channels.
 - Changed: [#5559](https://github.com/ethereum/aleth/pull/5559) Update peer validation error messages.
-- Fixed: [#5562](https://github.com/ethereum/aleth/pull/5562) Don't send header request messages to peers that haven't sent us Status yet.
 - Changed: [#5568](https://github.com/ethereum/aleth/pull/5568) Improve rlpx handshake log messages and create new rlpx log channel.
+- Changed: [#5576](https://github.com/ethereum/aleth/pull/5576) Moved sstore_combinations and static_Call50000_sha256 tests to stTimeConsuming test suite. (testeth runs them only with `--all` flag)
```
Comment (gumb0) [inline CHANGELOG.md]: (see https://keepachangelog.com/en/1.0.0/#how for changelog format)

**Comment by gumb0** [comment]
> net tests failing randomly
We know, see https://github.com/ethereum/aleth/issues/5544 and https://github.com/ethereum/aleth/pull/5581

**Pull request merged**